book/src/pallets/mosaic.md

```diff
@@ -22,37 +20,41 @@ decay function decayer, which depends on the penalty, `current_block`, and
 given by `budget - decayer(penalty, current_block, last_decay_block)`. The new 
 penalty is the decayed previous penalty plus the minted amount.
 
+## Workflow
 
-## Pallet Configuration
-
-### Event
-
-### PalletId
-
-### Assets
-
-### MinimumTTL
-
-Minimum time period, in blocks, that outgoing and incoming funds are locked.
-
-### BudgetPenaltyDecayer
-
-The budget penalty decayer.
+The Mosaic pallet is comprised of three main components: the Relayer interface, 
+incoming transactions, and outgoing transactions.
 
-### NetworkId
+### The Relayer Interface
 
-Network identifier.
+The Relayer interface provides the necessary functionality for managing the 
+constrains of the Relayer and exposing the functionality needed by the Relayer 
```

Comment:
constraints
```suggestion
constraints of the Relayer and exposing the functionality needed by the Relayer 
```

Reply:
32b481f76332c3f5a1fa991893aa48fa4193f22a

Reply:
Note, README and book/pallets synced here
617bec50d39cbbd1db640efacef83e65427980e1